pan3d/explorers/slice_explorer.py

```diff
@@ -0,0 +1,441 @@
+from trame.app import get_server
+from trame.decorators import TrameApp, change
+from trame.ui.vuetify3 import SinglePageWithDrawerLayout
+from trame.widgets import vuetify3 as v3, vtk as vtkw, html
+from pan3d.dataset_builder import DatasetBuilder
+import vtk, numpy as np
+
+colors = [
+    "Rainbow",       
+    "Inv Rainbow",   
+    "Greyscale",     
+    "Inv Greyscale", 
+]
+
+class LookupTable:
+    Rainbow = 0
+    Inverted_Rainbow = 1
+    Greyscale = 2
+    Inverted_Greyscale = 3
+
+# Color Map Callbacks
+def use_preset(sactor : vtk.vtkActor, dactor : vtk.vtkActor, preset : str) -> None:
```

Comment:
Wondering if we could have better preset defined in a json map or something.

Reply:
It was one of our plans to add some Divergent color maps similar to the ones defined here
https://github.com/Kitware/vtk-js/blob/master/Sources/Rendering/Core/ColorTransferFunction/LiteColorMaps.json

Reply:
Added the `LiteColorMaps`  from VTK.js